CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
 ## [Unreleased]
+- Use tags rather than time-based logs for information on requests - #5608 by @NyanKiyoshi
```

Comment:
Not sure we need to mention this as this will likely ship as part of 2.10

Reply:
This should be part of the changelogs of rc3